README.md

```diff
@@ -70,29 +70,30 @@ jobs:
     runs-on: ubuntu-latest
     name: Update README from json data
     steps:
-    - uses: actions/checkout@v2
-    - name: Read/Write data into README
-      uses: eddiejaoude/github-actions-reader-writer@v0.1
-      with:
-        json-file-path: 'data.json'
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        columns: 3
-        object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
-        file-to-use: 'README.md'
-        html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
+      - uses: actions/checkout@v2
+      - name: Read/Write data into README
+        uses: eddiejaoude/github-actions-reader-writer@v0.1
```

Comment:
I think we should probably update the links here.
```suggestion
        uses: EddieHubCommunity/gh-actions-html-table-generator@main
```

Reply:
https://github.com/EddieHubCommunity/gh-actions-html-table-generator/issues/31
There is no mention of the suggested change in the issue above.

Reply:
Right. Should probably be fixed in another PR.